clang/include/clang/Basic/DiagnosticCodeGen.h

```diff
@@ -0,0 +1,15 @@
+//===--- DiagnosticCodeGen.h - Diagnostics for libcodegen -------*- C++ -*-===//
```

Comment:
nit: We've updated the style guide to no longer include the file names in the header block.
https://llvm.org/docs/CodingStandards.html#file-headers

Reply:
Thanks for the heads up. I just copied an existing header not realizing the style guide had changed.